src/components/GetPOI.component.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import { FieldGroup } from '../components'
+import { Button } from 'react-bootstrap'
+
+export default class GetPOI extends Component {
+  state = { getPoiBy: '' }
+  handleTextChange = event => {
+    event.target.value.match(/^\d+(\.\d+)?$/) || event.target.value === ''

```

Comment:
whats going on here?

Reply:
limiting the textbox to just numbers with a regex

Reply:
oh typically you'd want a comment explaining something like that. but for this PR you dont actually need to have this at all